x/subspaces/ante/tx_checker.go

```diff
@@ -0,0 +1,48 @@
+package ante
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/ante"
+	antetypes "github.com/desmos-labs/desmos/v5/x/subspaces/ante/types"
+)
+
+// CheckTxFeeWithSubspaceMinPrices returns the tx checker that including the subspace allowed tokens into minimum prices list
+func CheckTxFeeWithSubspaceMinPrices(txFeeChecker ante.TxFeeChecker, sk antetypes.SubspacesKeeper) ante.TxFeeChecker {
+	return func(ctx sdk.Context, tx sdk.Tx) (sdk.Coins, int64, error) {
+		subspaceID, isSubspaceTx := GetTxSubspaceID(tx)
+		if !isSubspaceTx {
```

Comment:
@RiccardoM
Do you think we should exclude subspace management messages ( `MsgEditSubspace`, `MsgAddUserToUserGroup` and etc.), forcing admins mange subspace by using `DSM`?

Reply:
Yes, I believe we should *not* allow those messages to be paid using non-DSM tokens.

Reply:
I think it's best to do the opposite: adding a `ManageSubspaceMsg` interface that identifies only the one that should *not* be included. This is faster and easier to maintain for future features: it's going to be less likely to add new management operations rather than social operations

Reply:
Great idea, Implemented `ManageSubspaceMsg`, instead.